src/server/database/Database/Transaction.cpp

```diff
@@ -63,20 +67,51 @@ void Transaction::Cleanup()
     _cleanedUp = true;
 }
 
+const char* Transaction::ToString()
+{
+    std::string msg;
+    for (SQLElementData const& data : m_queries)
+    {
+        switch (data.type)
+        {
+        case SQL_ELEMENT_PREPARED:
```

Comment:
Indentation

Reply:
This is not the time to check indentation anyway

Reply:
@Jildor : `case` / `public` / `private` are labels and don't necessarily need to be indented.